src/Params.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
same as in the Calendar and Header components

Reply:
moved, and comment as in Calendar